packages/strapi-plugin-content-manager/config/queries/bookshelf.js

```diff
@@ -1,64 +1,38 @@
 const _ = require('lodash');
+const { convertRestQueryParams, buildQuery } = require('strapi-utils');
 
 module.exports = {
-  find: async function (params, populate, raw = false) {
-    return this.query(function(qb) {
-      _.forEach(params.where, (where, key) => {
-        if (_.isArray(where.value) && where.symbol !== 'IN' && where.symbol !== 'NOT IN') {
-          for (const value in where.value) {
-            qb[parseInt(value) ? 'orWhere' : 'where'](key, where.symbol, where.value[value]);
-          }
-        } else {
-          qb.where(key, where.symbol, where.value);
-        }
-      });
+  find: async function(params, populate) {
```

Comment:
You removed the `raw` condition, I added it for a specific reason, we should make a big tour of Strapi because I'm pretty sure something has been broken due to that.